server/src/logger.ts

```diff
@@ -0,0 +1,26 @@
+import pino from 'pino';
+import { elasticsearchURL, isProduction } from './config';
+
+const ecsFormat = require('@elastic/ecs-pino-format');
+const pinoElastic = require('pino-elasticsearch');
+const pinoms = require('pino-multi-stream');
+
+const pinoMultiStream = pinoms.multistream;
+
+const streamToElastic = pinoElastic({
```

Comment:
Only add `streamToElastic` if `elasticsearchURL` is not null. If `elasticsearchURL` is null then stream to stdout even if in production